.github/workflows/release.yml

```diff
@@ -76,3 +76,13 @@ jobs:
           script: |
             const script = require('./utils/update-latest-release.js')
             await script({ github, context })
+  summary:
+    name: Summary
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    if: always()
```

Comment:
Is this needed?

Reply:
Unsure, just copied it from the other workflow we use it in. I can nuke it.